gradle/bintray.gradle

```diff
@@ -38,64 +38,3 @@ bintray {
         }
     }
 }
-
-afterEvaluate { project ->
```

Comment:
I don't think that deploying to bintray will work without this?

Reply:
Same block is coming in from the maven deploy script.

Reply:
Awesome